client/packages/common/src/intl/locales/en/common.json

```
@@ -531,6 +531,7 @@
   "message.nothing-to-save": "Nothing to save",
   "messages.ago": "{{time}} ago",
   "messages.cannot-delete-multiple-lines": "One or more lines cannot be deleted",
+  "messages.cannot-view-vaccine-card": "This patient was enrolled in the {{ programName }} by another store and cannot be viewed in this store",
```

```suggestion
  "messages.cannot-view-vaccine-card": "This patient was enrolled in the {{ programName }} by another store and cannot be viewed in this store",
```

Something like... "This encounter cannot be viewed as the patient was enrolled in the {{ programName }} program by another store"